code/__DEFINES/color/lights.dm

```diff
@@ -34,4 +34,4 @@
 #define LIGHT_COLOR_INCANDESCENT_TUBE       "#E0EFF0" // rgb(224, 239, 240) Slightly blueish white.
 #define LIGHT_COLOR_INCANDESCENT_BULB       "#FFFEB8" // rgb(255, 254, 184) Slightly yellowish white.
 #define LIGHT_COLOR_INCANDESCENT_FLASHLIGHT "#FFCC66" // rgb(255, 204, 102) Slightly yellowish white.
-#define LIGHT_COLOR_NIGHTSHIFT              "#EFCC86" // rgb(239, 204, 134) Slightly yellowish white.
+#define LIGHT_COLOR_NIGHTSHIFT              "#616191" // rgb(134, 162, 239) Blueish white.
```

Comment:
The RGB value in the comment does not match the color here at all.
Additionally, the sheer darkness of this color affects the resulting light output in ways that might not be intentional; for instance, this is around 50% value, which means that the resulting light is, in fact, not increased to 0.6 power, but rather, effectively reduced to 0.3 (!!!) power. Light mixing might also be an issue with a light color this dark, though admittedly I can't recall which lighting backends play well with dark light colors

Reply:
Seems to function well when it comes to the testmerge as well as locally. Comment/RGB was fixed